### DISCUSSION TITLE
Is there a mobile version of github?

### DISCUSSION THREAD

**Discussion by vfernando01** [community/community#62518] [Mobile]
### Select Topic Area

Question

### Body

Is there a mobile version of github?

**Comment by franciscusvalentinus** [comment]
yes

**Comment by mayachee** [comment]
yes

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
